### ISSUE TITLE
Bloc pattern requested

### ISSUE THREAD

**Issue by knightcube** [nityanandagohain/in-time#5]
Since the app is in its very early stages currently, it would be a good idea to follow a particular architecture while developing the codebase. I think Bloc would be good for everyone.

**Comment by nityanandagohain** [comment]
Yes, that will be good. You can add that information in the readme.

**Comment by knightcube** [comment]
Yeah, it will require a few more changes in the project structure as well. I will make those changes and submit the PR once #7 is reviewed and merged.

**Comment by nityanandagohain** [comment]
Sure .
